policyengine_us/parameters/gov/states/co/tax/income/additions/co_itemized_or_standard_deductions/limits/head_of_household.yaml

```diff
@@ -0,0 +1,17 @@
+description: Colorado issues itemized or standard deduction for head of household filers above this limit.
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
+  unit: currency-USD
+  label: Colorado itemized or standard deduction head of household filers limit
+  reference:
+  - title: C.R.S. 39-22-104 . Income tax imposed on individuals, estates, and trusts - section (3)(p) - (p.5)
+    href: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=0ac0632f-7202-4eec-b761-6e36fad6542d&action=pawlinkdoc&pdcomponentid=&pddocfullpath=%2fshared%2fdocument%2fstatutes-legislation%2furn%3acontentItem%3a683G-JJ73-CGX8-04HR-00008-00&pdtocnodeidentifier=ABPAACAACAABAACAAF&config=014FJAAyNGJkY2Y4Zi1mNjgyLTRkN2YtYmE4OS03NTYzNzYzOTg0OGEKAFBvZENhdGFsb2d592qv2Kywlf8caKqYROP5&ecomp=k2vckkk&prid=713552e7-276b-4575-a579-ea86a2ff3c84
```

Comment:
could you add a casetext or justia reference instead of lexis -> lexis seems to have some problems with stability